docs/DebuggerHTML-Components-Reducers-Actions.md

```diff
@@ -0,0 +1,663 @@
+Debugger.html
+=============
+
+Debugger.html is an open source project that is built on top of React and Redux that functions as a standalone debugger for Firefox, Chrome and Node.  The project is being tested currently in Firefox Nightly and is available in the DevTools debugger tab when the preference devtools.debugger.new-debugger-frontend is enabled. This project is being created to provide a debugger that is stand-alone and does not require a specific browser tool to do debugging.
```

Comment:
I would take this section out and show it as a note / quote below this paragraph:

> The project is being tested currently in Firefox Nightly and is available in the DevTools debugger tab when the preference `devtools.debugger.new-debugger-frontend` is enabled. 

It might also make sense to have another separate doc just for the "How to enable on Nightly" which we can link to for full instructions and delete once it is on by default.